common/src/main/java/io/novafoundation/nova/common/utils/KotlinExt.kt

```diff
@@ -48,6 +48,11 @@ inline fun <T, R> Result<T>.flatMap(transform: (T) -> Result<R>): Result<R> {
     )
 }
 
+inline fun <T, R> Result<T>.flatMapCatching(transform: (T) -> Result<R>): Result<R> {
```

Comment:
I use it here to avoid an exception in: `session.getSavedPassword().getOrThrow()`
```
suspend fun CloudBackupService.fetchAndDecryptExistingBackupWithSavedPassword(): Result<CloudBackup> {
    return fetchBackup()
        .flatMapCatching { encryptedBackup ->
            val password = session.getSavedPassword().getOrThrow()
            encryptedBackup.decrypt(password)
        }
}

```